src/main/webapp/app/exercises/text/assess/textblock-feedback-editor/textblock-feedback-editor.component.ts

```diff
@@ -5,25 +5,10 @@ import { ConfirmIconComponent } from 'app/shared/confirm-icon/confirm-icon.compo
 import { StructuredGradingCriterionService } from 'app/exercises/shared/structured-grading-criterion/structured-grading-criterion.service';
 import { FeedbackConflictType } from 'app/entities/feedback-conflict';
 import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
-import { TextAssessmentService } from 'app/exercises/text/assess/text-assessment.service';
-import { StudentParticipation } from 'app/entities/participation/student-participation.model';
 import { ActivatedRoute } from '@angular/router';
-import { lastValueFrom } from 'rxjs';
 import { TextAssessmentEventType } from 'app/entities/text-assesment-event.model';
 import { TextAssessmentAnalytics } from 'app/exercises/text/assess/analytics/text-assesment-analytics.service';
-import {
-    faAngleRight,
-    faBalanceScaleRight,
-    faEdit,
-    faExclamation,
-    faExclamationTriangle,
-    faInfoCircle,
-    faQuestionCircle,
-    faRobot,
-    faSearch,
-    faTimes,
-    faTrash,
-} from '@fortawesome/free-solid-svg-icons';
+import { faAngleRight, faBalanceScaleRight, faEdit, faExclamation, faExclamationTriangle, faQuestionCircle, faRobot, faTimes, faTrash } from '@fortawesome/free-solid-svg-icons';
```

Comment:
You can also remove the `listOfBlocksWithFeedback` property of the component.

Reply:
Thanks! I removed it :)